templates/falco_rules.local.yaml.epp

```diff
@@ -0,0 +1,40 @@
+<%- | Array[Hash] $local_rules
+| -%>
+##############################################
+# THIS FILE MANAGED BY PUPPET. DO NOT MODIFY #
+##############################################
+
+#
+# Copyright (C) 2019 The Falco Authors.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+####################
+# Your custom rules!
+####################
+
+# Add new rules, like this one
+# - rule: The program "sudo" is run in a container
+#   desc: An event will trigger every time you run sudo in a container
+#   condition: evt.type = execve and evt.dir=< and container.id != host and proc.name = sudo
+#   output: "Sudo run in container (user=%user.name %container.info parent=%proc.pname cmdline=%proc.cmdline)"
+#   priority: ERROR
+#   tags: [users, container]
+
+# Or override/append to any rule, macro, or list from the Default Rules
+
+<% unless $local_rules.empty() { %>
+<%= $local_rules.to_yaml({'line_width' => -1}).regsubst('^---', '') %>
```

Comment:
Why _are_ you removing the `---` (yaml start of document) here?  Most yaml linters would moan about this.

Perhaps you don't even need the conditional at all??  If $local_rules is empty, then `to_yaml` should correctly output
```yaml
--- []
```

Reply:
I am removing it because it is not needed and is not part of the original file format - I want to keep the formatting true to how it looked before being managed.